basin-helper

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+PATH_TO_JSON=/tmp/basin.json
+PATH_TO_FILES=$1
+PATH_TO_SHARD=$2
+
+echo $PATH_TO_FILES $PATH_TO_SHARD
+
+basin-processor $PATH_TO_FILES $PATH_TO_JSON
```

Comment:
In order to make this a bit more friendly you should abort if the wrong number of arguments are passed, and also abort if one of these tools exits with a nonzero exit code.